frontend/workflows/projectCatalog/src/details/index.tsx

```diff
@@ -142,7 +142,7 @@ const Details: React.FC<ProjectDetailsWorkflowProps> = ({ children, chips }) =>
               </Grid>
             )}
           </Grid>
-          <Grid container direction="row" spacing={2}>
+          <Grid container direction="row" spacing={1}>
```

Comment:
why change the spacing here?

Reply:
When I was originally going through it the spacing seemed off so I adjusted it to try and match v4. Can't tell a difference anymore..so putting it back :)